cardano-chain-gen/src/Cardano/Mock/Forging/Tx/Conway.hs

```diff
@@ -483,6 +484,21 @@ mkDummyTxBody =
     mempty
     (Withdrawals mempty)
 
+mkDummyTxBodyWithCoin ::
```

Comment:
You may want to specify that you're adding a fee, since there are multiple fields that take `Coin`s
```suggestion
mkDummyTxBodyWithFee ::
```